cedar-policy-core/src/ast/expr.rs

```diff
@@ -530,13 +530,16 @@ impl Expr {
                 arg1.substitute(definitions)?,
                 arg2.substitute(definitions)?,
             )),
-            ExprKind::ExtensionFunctionApp { op, args } => {
+            ExprKind::ExtensionFunctionApp {
+                function_name,
```

Comment:
[nit] Personally I prefer longer names but to be consistent with internal style I think this should be "fn_name".
Otherwise, maybe rename "args" to "arguments".

Reply:
Done.